invenio_cli/commands/services.py

```diff
@@ -164,36 +165,43 @@ def _setup(self):
                 env={"PIPENV_VERBOSITY": "-1"},
                 message="Creating indices...",
             ),
-            CommandStep(
-                cmd=[
-                    "pipenv",
-                    "run",
-                    "invenio",
-                    "rdm-records",
-                    "custom-fields",
-                    "init",
-                ],
-                env={"PIPENV_VERBOSITY": "-1"},
-                message="Creating custom fields for records...",
-            ),
-            CommandStep(
-                cmd=[
-                    "pipenv",
-                    "run",
-                    "invenio",
-                    "communities",
-                    "custom-fields",
-                    "init",
-                ],
-                env={"PIPENV_VERBOSITY": "-1"},
-                message="Creating custom fields for communities...",
-            ),
+        ]
+
+        if rdm_version[0] < 10:
```

Comment:
```suggestion
        if rdm_version[0] >= 10:
```
?

What if we do something more structured, e.g. create a enum/map of features with version, for example (allow me pseudo-code):
```
versions = { 
  10: [custom-fields, administration],
   ...
}
```
then an helper:
```
get_enabled_features():
  return versions[current-app-rdm version]
```

and then in the code we get that map and use it?
```
if custom-fields-in-version:
  run init command
```